### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by generationmake**

### CODE REVIEW

**Review by aentinger — COMMENTED** [review]
Looking good so far ;)

**Comment by generationmake** [comment]
Hi @aentinger 
Could you please test the function of the code? Thanks

File: CyphalRobotController07-CAN-firmware.ino [hunk 1]
```diff
@@ -43,6 +43,18 @@ static uint8_t const EEPROM_I2C_DEV_ADDR = 0x50;
 
 static int const MCP2515_CS_PIN  = 17;
 static int const MCP2515_INT_PIN = 20;
+static int const EM_STOP_PIN           = 12;
+static int const OUTPUT_0_PIN          = 21; /* GP21 */
+static int const OUTPUT_1_PIN          = 22; /* GP22 */
+static int const ANALOG_INPUT_0_PIN = 26;
+static int const ANALOG_INPUT_1_PIN = 27;
+static int const ANALOG_INPUT_2_PIN = 28;
+#define MOT0_1 9
```
Comment (aentinger) [inline CyphalRobotController07-CAN-firmware.ino]: Use `static int const` instead of `#define`.

Comment (generationmake) [reply CyphalRobotController07-CAN-firmware.ino]: yes, i will change it.

But are the motors moving?

**Comment by generationmake** [comment]
motor can be controlled and is spinning.
will merge

**Pull request merged**